cli/cook/subcommands/kill.py

```diff
@@ -85,20 +76,25 @@ def kill_entities(query_result, clusters):
     succeeded = []
     clusters_by_name = {c['name']: c for c in clusters}
 
-    def __kill(cluster, uuids, kill_fn):
+    def __kill(cluster, uuids, kill_fn, entity_type):
         if len(uuids) > 0:
             for uuid_batch in partition(uuids, kill_batch_size):
                 success = kill_fn(cluster, uuid_batch)
                 (succeeded if success else failed).extend(uuid_batch)
+                for uuid in uuid_batch:
```

Comment:
With my previous comment I was expecting the prints to come all at once in the end categorized by succeeded and failed after the `__kill` calls below. The current printing happens in the same manner in which they were happening when they were inside `__kill_entities`

Reply:
Ah, I misunderstood your previous comment. What's the downside of having them inside the `__kill` chain versus after it?

Reply:
I'm okay with them being inside `__kill` but outside the partition loop as then the prints are grouped by entity type first and then success/failure. My motivation was that I wanted to see all the success prints and then all the failed prints as the failed ones are the ones I would care about.  If they are interleaved, then I have to go searching for the ones that failed.

Reply:
Done